Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Private/InteropCodeGenerator/TypeBindingGenerator.cpp

```diff
@@ -551,6 +556,20 @@ void GeneratePropertyToUnrealConversion(FCodeWriter& Writer, const FString& Upda
 	}
 }
 
+void GetPropertyText(FStringOutputDevice& PropertyText, UProperty* Prop)
+{
+	FString* ActualCppType = nullptr;
+	FString ObjectCppType;
```

Comment:
Move this inside the if state ment

Reply:
It would go out of scope when ActualCppType needs to be able to point to it.

Reply:
Sorry my bad, overlooked that.